### DISCUSSION TITLE
トラックボール基盤が奥まで入らない

### DISCUSSION THREAD

**Discussion by shimizu432** [Yowkees/keyball#174] [Q&A] [answered]
トラックボールの読み取り基盤が、おそらく奥までしっかり入りません
ガイドにあるように抜き挿しして調整してみても変わりませんし、無理をしようとすると負荷がかかりそうで怖い状態です
![20230104_230840](https://user-images.githubusercontent.com/22129990/210936087-abad5dc1-81b5-4698-9a1c-d74960233bad.jpg)

パーツを斜めにはんだづけしてしまっていたことが原因かと思い、吸い取り線や吸い取り器での修正を試みたのですが、うまく吸い出せませんでした
![20230106_141342](https://user-images.githubusercontent.com/22129990/210936909-dfd05f3b-2f18-4ba4-bc9b-7bb2e2fe143e.jpg)

適切な修正方法をご教示いただけるでしょうか？

**Comment by shimizu432** [comment, marked as answer]
自己解決しました
基盤が引っかかるツメの部分を少しヤスりました
![20230106_162353](https://user-images.githubusercontent.com/22129990/210955957-be8903a3-ac4c-43da-8cd2-fc116f1493b8.jpg)

**Comment by Yowkees** [comment]
やすりがけにより解決していただき、ご対応ありがとうございました。
トラックボールの動作に問題がないようでしたら、今後も使用可能かと思います。
今後ともどうぞよろしくお願いいたします。
